scripts/dependencies/common_tools.sh

```diff
@@ -34,36 +34,37 @@ function waitForSuccess() {
     done
 }
 
-# Installs an operator according to given subscription resource and waits, params: operatorSubscriptionResource
+# Installs an operator according to given subscription resource and waits, params: operatorSubscriptionResource, namespace (defaults to openshift-operators)
```

Comment:
Added this argument as serverless operator is no longer installed in `openshift-operators` namespace anymore.